modules/core/assets/configs/menuBackgroundShipConfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Menu Ships": [
+    "imperialTiny",
+    "imperialSmall",
+    "minerSmall",
+    "minerBoss",
+    "desertSmall",
+    "desertOrbiter"
```

Comment:
Please ensure the module is added to these. A module should be allowed to add a ship from another module. You'd also have to handle potential duplicates :+1:

Reply:
Oh, I was wondering if I should do that, but when you said 
> That way a module can add its own ships if it so desires.

I thought modules would only be able to add their own ships :P, I'll change it now